internal/rows/rows.go

```diff
@@ -68,6 +68,7 @@ var _ driver.RowsColumnTypeLength = (*rows)(nil)
 var _ dbsqlrows.Rows = (*rows)(nil)
 
 func NewRows(
+	ctx context.Context,
```

Comment:
If the only thing we are using ctx for is to extract a logger.  I would rather pass in a logger as the last argument to NewRows and ensure that dbsqllog.AddContext handles a nil first argument.